docs/build/specify-observables-pauli.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Specify observables in the Pauli basis
+description: Measure circuits in different Pauli bases, which is required to measure observables that are not diagonal in the computational basis.
+
+---
+
+# Specify observables in the Pauli basis
```

Comment:
For the specify section are there any other code snippets we can add in? Ultimately we want to show off what qiskit can do here, so if there are any other code snippets that could be added that would be ideal. It would also help break up the text a bit as right now the first section feels a bit like a wall of text and equations

Reply:
I added another one on operator construction. A more in-depth site could also go in detail of how Paulis are represented in Qiskit with different classes, maybe we could add that as follow-up